cmd/esctl/main.go

```diff
@@ -49,9 +51,16 @@ func setup() {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+	if len(os.Getenv("ESCTL_GEN_DOCS")) > 0 {
```

Comment:
Can you add a Makefile rule to do this. This will have to be run every time a new (sub)command is added right?

Reply:
yes we will have to do that, will add a makefile target for this